### ISSUE TITLE
Upgrade breaks nodes

### ISSUE THREAD

**Issue by runningman84** [continuousphp/aws-codedeploy-agent#8]
Upgrade fails, we used to use the old 0.4.x release of this cookbook. Afer an update to the latest 0.6 release all of our nodes fail:

```
[2016-09-20T15:48:21+00:00] INFO: Processing link[/opt/codedeploy-agent] action create (aws-codedeploy-agent::default line 15)
[2016-09-20T15:48:21+00:00] INFO: Running queued delayed notifications before re-raising exception
[2016-09-20T15:48:21+00:00] INFO: ruby_block[delay filebeat service start] sending start action to service[filebeat] (delayed)
[2016-09-20T15:48:21+00:00] INFO: Processing service[filebeat] action start (filebeat::config line 61)
[2016-09-20T15:48:21+00:00] ERROR: Running exception handlers
[2016-09-20T15:48:21+00:00] ERROR: Exception handlers complete
[2016-09-20T15:48:21+00:00] FATAL: Stacktrace dumped to /var/chef/cache/chef-stacktrace.out
[2016-09-20T15:48:21+00:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
[2016-09-20T15:48:21+00:00] ERROR: link[/opt/codedeploy-agent] (aws-codedeploy-agent::default line 15) had an error: Errno::EISDIR: Is a directory @ unlink_internal - /opt/codedeploy-agent
[2016-09-20T15:48:21+00:00] ERROR: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
```

**Comment by oswaldderiemaecker** [comment]
Please tell me which OS/Version ?

Are you upgrading a node which has been installed with the version 0.4.x with the version 0.6.0 or are you packing a new AMI, if so from which base AMI ?

**Comment by runningman84** [comment]
We are using a Ubuntu 14.04 ami. Our nodes have installed code deploy using your cookbook. Once we did a berkshelf update and upgrade our Dev environment broke.

**Comment by oswaldderiemaecker** [comment]
@runningman84 a new version of the cookbook has been released (0.7.0), please ensure your current version is pinned and test the new release before upgrading your dev environment. Thank you.
